karma.conf.js

```diff
@@ -16,6 +16,7 @@ module.exports = function (config) {
             'karma-phantomjs-launcher',
             'karma-coverage'
         ],
+        usePolling: true,
```

Comment:
What does `usePolling` do? I don't see it in their docs: http://karma-runner.github.io/0.13/config/configuration-file.html